README.md

```diff
@@ -1 +1,263 @@
+
+## 2.2 Raw (Noncanonicalized) Data
+
+Canonicalization is not required if data is encoded as raw bytes (multicodec `0x55`). The exact bytes are already present, and MUST not be changed.
+
+# 3 Varsig Format
+
+After being decoded from Varints, a varsig includes the following segments:
+
+| Segment Name              | Type          | Description                                     | Required |
+|---------------------------|---------------|-------------------------------------------------|----------|
+| Varsig Multiformat Prefix | `0x34`        | The multicodec varsig prefix                    | Yes      |
+| Content Multicodec Prefix | `Multiformat` | The IPLD encoding uses to canonicalize the data | Yes      |
+| Public Key Prefix         | `Multicodec`  | The multicodec prefix for the public key type   | Yes      |
+| Multihash                 | `Multihash`   | The multicodec prefix for the multihash         | Yes      |
+| Raw Signature             | `Bytes`       | The raw signature                               | Yes      |
+  
+## 3.1 Segments
+
+### 3.1.1 Varsig Prefix
+
+The varsig prefix MUST be `0x34`.
+
+### 3.1.2 Public Key Prefix
+
+The [multicodec](https://github.com/multiformats/multicodec) prefix for the public key type associated with the signature.
+
+For example: `0xe7` for secp256k1, or `0x1205` for RSA.
+
+### 3.1.3 Hash Prefix
+
+The [multicodec](https://github.com/multiformats/multicodec) for the [multihash](https://github.com/multiformats/multihash) used by the signature. This is the first segment of a .
+
+For example: `0x16` for SHA3-256, or `0x1e` for BLAKE3
+
+### 3.1.4 Content Multicodec Prefix
+
+The [multicodec](https://github.com/multiformats/multicodec) prefix of the encoding scheme used.
+
+For example: `0x55` for raw bytes (no special encoding), `0x0129` for DAG-JSON, and `0x70` for DAG-PB.
+
+### 3.1.5 Raw Signature
+
+The raw signature bytes
+
+## 3.2 Segment Layout
+
+```xml
+<multiformat 0x34><varint content_multicodec><varint key_multicodec><varint multihash><varint raw_signature>
+```
+
+## 3.3 Example
+
+Varsig: `0x68d204da0324deb7de9af1d9a2a302`
+
+|               | Multiformat | Content Encoding | Public Key Algorithm | Hash Algorithm | Signature              |
+|---------------|-------------|------------------|----------------------|----------------|------------------------|
+| Semantic Type | Varsig      | DAG-JSON         | EdDSA                | SHA2-256       | Bytes                  |
+| Value         | `0x34`      | `0x0129`         | `0xed`               | `0x12`         | `0x123456789ABCDEF`    |
+| Varint        | `0x68`      | `0xd204`         | `0xda03`             | `0x24`         | `0xdeb7de9af1d9a2a302` |
```

Comment:
I'm confused about what this line is showing

Reply:
The last line shows the data encoded as varints, which have a different low-level representation than the numbers that they encode (LEB128)

Reply:
Added more descriptive text above and clarified the rows